### DISCUSSION TITLE
There is no init command

### DISCUSSION THREAD

**Discussion by P-Asta** [gleam-lang/gleam#2741] [Ideas & suggestions]
It is strange that there is a new command but no init command. Please make it

**Comment by lpil** [comment]
The new command can already do what the init command does, assuming you mean npm init.

**Reply by P-Asta** [reply]
`gleam new` exists, but `gleam init` does not exist

**Reply by lpil** [reply]
What is it that you are unable to do with the gleam new command? I'm very happy to help you resolve your issue once I know the details. Thank you.
